delta/sdk/src/test/scala/ch/epfl/bluebrain/nexus/delta/sdk/projects/ScopeInitializationErrorStoreSuite.scala

```diff
@@ -0,0 +1,68 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.projects
+
+import cats.effect.IO
+import ch.epfl.bluebrain.nexus.delta.sdk.error.ServiceError.ScopeInitializationFailed
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.{EntityType, ProjectRef}
+import ch.epfl.bluebrain.nexus.delta.sourcing.postgres.Doobie
+import ch.epfl.bluebrain.nexus.testkit.clock.MutableClock
+import ch.epfl.bluebrain.nexus.testkit.mu.NexusSuite
+import doobie.implicits._
+import doobie.postgres.implicits._
+import munit.AnyFixture
+
+import java.time.Instant
+
+class ScopeInitializationErrorStoreSuite
+    extends NexusSuite
+    with MutableClock.Fixture
+    with Doobie.Fixture
+    with Doobie.Assertions {
+
+  override def munitFixtures: Seq[AnyFixture[_]] = List(doobie, mutableClockFixture)
+  private lazy val xas                           = doobie()
+  private lazy val mutableClock: MutableClock    = mutableClockFixture()
+
+  private lazy val errorStore = ScopeInitializationErrorStore(xas, mutableClock)
+
+  private val project        = ProjectRef.unsafe("org", "proj")
+  private val entityType     = EntityType("test")
+  private val scopeInitError = ScopeInitializationFailed("boom")
+
+  private val scopeInitErrorRow = ScopeInitErrorRow(
+    1,
+    entityType.value,
+    project.organization.value,
+    project.project.value,
+    scopeInitError.reason,
+    Instant.EPOCH
+  )
+
+  test("Inserting an error should succeed") {
+    assertStoreIsEmpty >>
```

Comment:
Would it ever not be empty?

Reply:
If I were to add another test now, it would start with a non empty store. Should it be more like a `resetStore` method, here? Not sure

Reply:
I think the tests having a clean slate isn't really something you need to check at the beginning of the test, it's more for a beforeAll